Description: Fix build problem with libpng16.
 Backport fix from upstream to use sizeof() instead of png_sizeof().
Author: Tristan Matthew <le.businessman@gmail.com>
Bug-Debian: https://bugs.debian.org/809949
Origin: backport, https://git.xiph.org/?p=theora.git;a=commitdiff;h=7288b539c52e99168488dc3a343845c9365617c8

Index: libtheora/examples/png2theora.c
===================================================================
--- libtheora.orig/examples/png2theora.c	2016-02-07 10:37:24.586795794 +0000
+++ libtheora/examples/png2theora.c	2016-02-07 10:38:11.983361108 +0000
@@ -462,9 +462,9 @@
   png_set_strip_alpha(png_ptr);
 
   row_data = (png_bytep)png_malloc(png_ptr,
-    3*height*width*png_sizeof(*row_data));
+    3*height*width*sizeof(*row_data));
   row_pointers = (png_bytep *)png_malloc(png_ptr,
-    height*png_sizeof(*row_pointers));
+    height*sizeof(*row_pointers));
   for(y = 0; y < height; y++) {
     row_pointers[y] = row_data + y*(3*width);
   }
